TestApp/app/src/main/java/com/example/testapp/ui/LoginActivity.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.testapp.ui
+
```

Comment:
add a sperate package for login functionalities in ui . ui.login

Reply:
added